site/content/contribute/more-info/server/developer-setup/centos.md

```diff
@@ -32,7 +32,7 @@
     sudo chmod +x /usr/local/bin/docker-compose
     ```
 
-3. Install Go (modify installation to latest Go version from https://golang.org/dl/):
+3. Install Go (modify installation to latest Go version from <https://golang.org/dl/>):
```

Comment:
Here it fixes a broken link to golang download page (it was my initial issue, but it gave me the opportunity to check all the links in this page)